tests-integration/src/test/java/com/contentful/vaultintegration/LocalizedLinksTest.java

```diff
@@ -0,0 +1,37 @@
+package com.contentful.vaultintegration;
+
+import com.contentful.vault.Vault;
+import com.contentful.vaultintegration.lib.demo.DemoSpace;
+import com.contentful.vaultintegration.lib.demo.Dog;
+import com.contentful.vaultintegration.lib.demo.Dog$Fields;
+import java.util.List;
+import org.junit.Test;
+import org.robolectric.RuntimeEnvironment;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public final class LocalizedLinksTest extends BaseTest {
+  @Override protected void setupVault() {
+    vault = Vault.with(RuntimeEnvironment.application, DemoSpace.class);
```

Comment:
I'm actually gonna move this out to it's own space so we can always use the original demo space data.